src/deckgl-layers/layer-utils/utils.js

```diff
@@ -84,9 +82,18 @@ export function getColorScaleFunction(layer) {
     .range(layer.props.colorRange);
 }
 
+export function getElevationScaleFunction(layer) {
+  const elevationRange = layer.props.elevationRange;
+  const elevationDomain = layer.props.elevationDomain || layer.state.elevationValueDomain;
+
+  layer.state.sizeScaleFunc = getScaleFunctor(layer.props.sizeScale)()
```

Comment:
This is confusing because `deck.gl` uses `elevationScale` to reference a scalar value that each height is multiplied by.

Most other places in Kepler `___Scale` refers to one of the `SCALE_TYPES`. Is there any way we can explicitly call out this difference?

Reply:
__scale for value calculation in kepler.gl layer is stored in layer.config

__scale passed to deck.gl as multiplier in is store in layer.config.visConfig.

So at least they won't override each other

You can us `scaleScleType` or `elevationScaleType` as layer props to pass it to deck.gl